xbmc/pvr/addons/PVRClients.cpp

```diff
@@ -1325,7 +1325,7 @@ bool CPVRClients::UpdateAddons(void)
     // You need a tuner, backend software, and an add-on for the backend to be able to use PVR.
     // Please visit http://kodi.wiki/view/PVR to learn more.
     m_bNoAddonWarningDisplayed = true;
-    CGUIDialogOK::ShowAndGetInput(19271, 19272, 19273, 19274);
+    CGUIDialogOK::ShowAndGetInput(19271, 19272);
```

Comment:
Something went wrong here. You moved the entire description to the dialog headline 19271, labels 19272 and 19273 have been deleted and 19274 is still there in strings.po